txn.go

```diff
@@ -513,7 +513,14 @@ func (txn *Txn) DeleteAll(table, index string, args ...interface{}) (int, error)
 }
 
 // FirstWatch is used to return the first matching object for
-// the given constraints on the index along with the watch channel
+// the given constraints on the index along with the watch channel.
+//
+// Note that returned value will reflect snapshot from when txn was created.
```

Comment:
```suggestion
// Note that all values read in a single transaction form a consistent snapshot from the time
// when the transaction was created.
```
This read a little more naturally to me although I think what you have is probably clear enough to most people!

Reply:
Thanks! I'll just modify "_a single transaction_" to "_the transaction_" to make it clearer that we refer to that one `txn` on which the `FirstWatch` method (and others) are called, not just arbitrary single transaction.